etc/desi.json

```diff
@@ -6,6 +6,17 @@
     },
     "datachallenge":{
         "exclude":[],
+        "dc17a-twopct":{
+            "dc17a-twopct/[^/]+$":"dc17a-twopct/toplevel.tar",
+            "dc17a-twopct/batch/.*$":"dc17a-twopct/batch.tar",
+            "dc17a-twopct/fiberassign/.*$":"dc17a-twopct/fiberassign.tar",
+            "dc17a-twopct/spectro/redux/([0-9a-zA-Z_\\-]+)/exposures/([0-9]){8}/.*$":"dc17a-twopct/spectro/redux/\\1/exposures/\\2.tar",
```

Comment:
You don't need to escape a hyphen when it's not in between anything.

Also `([0-9]){8}` is incorrect.  It should be `([0-9]{8})`.